packages/expo/android/src/main/java/expo/modules/ReactNativeHostWrapperBase.kt

```diff
@@ -73,6 +74,7 @@ open class ReactNativeHostWrapperBase(
       .firstOrNull() ?: host.useDeveloperSupport
   }
 
+  // TODO vonovak could be removed?
```

Comment:
for these two TODOs, i think we should still keep them. when people override  `getJSMainModuleName` or `getPackages` in MainApplication. we should make sure we call their overridden methods. remove these two methods will just the the default methods from `DefaultReactNativeHost`.